build.gradle.kts

```diff
@@ -27,6 +32,14 @@ dependencies {
     compileOnly(libs.ini4j)
     compileOnly(libs.kotlinxSerialization)
     testImplementation(kotlin("test"))
+//    implementation("com.redhat.devtools:lsp4ij:0.10.0")
+    implementation("org.eclipse.lsp4mp:org.eclipse.lsp4mp.ls:0.10.0") {
```

Comment:
You dont need lsp4mp which ptovide a microprofile ls and dont need qute ls

Reply:
Thank you for your comment.
I just removed it.